app.rb

```diff
@@ -1,8 +1,20 @@
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  # get '/' do
+    # redirect '/chitter/peeps'
+  # end
+
+  get '/chitter/peeps' do
```

Comment:
It looks like you tried to organise your routes, however the best way to do this is with a system we call REST. For example with your routes as they are now you would presumably have /chitter/ in front of everything making it unhelpful bloat, and peeps is a bit generic and doesn't give any indication of what that route is for. Perhaps something like get '/peeps' for the landing page and then post '/peeps/new' might be a better solution for your other route. This way we know that these routes are for peeps and that one of them is to create a new one.